### PR TITLE
MeB takeover

### PR DESCRIPTION
**Pull request by akshaaatt**
- Migrate the design-system to be compatible for all MeB projects.

### CODE REVIEW

File: dist/bootstrap.css [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
  * In order to not break parts of the website that haven't been converted,
- * all Bootstrapified components must have a .bs class on their containing
+ * all Bootstrapified shared must have a .bs class on their containing
```
Comment (mwiencek) [inline dist/bootstrap.css:3]: This change seems wrong

File: scss/bootstrap-prod.scss [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /*
  * In order to not break parts of the website that haven't been converted,
- * all Bootstrapified components must have a .bs class on their containing
+ * all Bootstrapified shared must have a .bs class on their containing
```
Comment (mwiencek) [inline scss/bootstrap-prod.scss:3]: Same here

**Review by mwiencek — CHANGES_REQUESTED** [review]

**Comment by y-young** [comment]
What about putting shared (common) components under `components` and making multiple folders for exclusive components of projects? e.g: `components/mb`, `components/lb`.

File: dist/bootstrap.css (hunk 1 above)
Comment (akshaaatt) [reply dist/bootstrap.css:3]: Thanks for pointing this out!

File: scss/bootstrap-prod.scss (hunk 2 above)
Comment (akshaaatt) [reply scss/bootstrap-prod.scss:3]: Thank you!

**Comment by akshaaatt** [comment]
> What about putting shared (common) components under `components` and making multiple folders for exclusive components of projects? e.g: `components/mb`, `components/lb`.

@y-young Thanks for the thought. I was going to name them as `common` but went with `shared` so that the hierarchy can be maintained and common doesn't nudge up and confuse us. Shared will be down in the folder and easy to access for the developers who work on this in the future. 
Moving shared under components would work but then they would be handled separately. Currently they can be in the same folder. If these are necessary issues and could be really fruitful to update, we can consider doing so.

**Pull request merged**